bundle/compliance/cis_gcp/rules/cis_6_1_2/rule.rego

```diff
@@ -0,0 +1,20 @@
+package compliance.cis_gcp.rules.cis_6_1_2
+
+import data.compliance.lib.common
+import data.compliance.policy.gcp.data_adapter
+
+finding = result {
+	data_adapter.is_cloud_sql
+	data_adapter.is_cloud_my_sql
+
+	result := common.generate_result_without_expected(
+		common.calculate_result(is_local_infile_flag_disabled),
+		data_adapter.resource,
+	)
+}
+
+is_local_infile_flag_disabled {
+	flags := data_adapter.resource.data.settings.databaseFlags[_]
+	flags.name == "skip_show_database"
+	flags.value == "off"
```

Comment:
> Ensure the below command returns `on` for every Cloud SQL Mysql database instance

Shouldn't it be the other way around?

Reply:
yes, thanks for catching this. mixed up with 6.1.3